cmd/antrea-controller/controller.go

```diff
@@ -50,6 +50,7 @@ func run(o *Options) error {
 	podInformer := informerFactory.Core().V1().Pods()
 	namespaceInformer := informerFactory.Core().V1().Namespaces()
 	networkPolicyInformer := informerFactory.Networking().V1().NetworkPolicies()
+	nodeInformer := informerFactory.Core().V1().Nodes()
```

Comment:
@tnqn: do we have a way to watch only specific fields instead of all changes?

Reply:
I think no, I never saw client options to specify required fields or server code to fill them. Even `kubectl get pods -o custom-columns=` gets the whole object from server and gets specified fields on client side.